scripts/refresh_trends.py

```
@@ -0,0 +1,262 @@
+                    "tops', 'loafers'). Lowercase."
+                ),
+                "maxItems": 5,
+            },
+        },
+        "required": ["key_colors", "key_silhouettes", "key_pieces"],
+    },
+}
+
+
+def iso_monday(today: date | None = None) -> date:
+    today = today or date.today()
```

Ruff (0.15.12)

[warning] `datetime.date.today()` used

(DTZ011)